packages/mobile/src/pincode/PincodeSet.tsx

```diff
@@ -104,6 +109,7 @@ export class PincodeSet extends React.Component<Props, State> {
       this.props.setPincode(PincodeType.CustomPin)
       ValoraAnalytics.track(OnboardingEvents.pin_set)
       this.navigateToNextScreen()
+      Logger.showMessage('PIN changed')
```

Comment:
During onboarding we don't want this toast to show up. Also, it should be translated :)

Reply:
thanks! added conditional